scripts/install_dependencies

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
Can we delete this since we're installing deps in other scripts?

Reply:
It's a place holder for AWS codeDeploy hooks, AWS codeDeploy's behavior is inconsistent when we don't specify these hooks, hence i left it even though it's empty. Will also serve as a place to do any other env initialization like update update some agent etc if required.